docs/src/docs/quick-start.md

```diff
@@ -125,20 +123,24 @@ Now unpack an LLM ModelKit - we have [several](https://github.com/orgs/jozu-ai/p
 kit unpack ghcr.io/jozu-ai/phi3:3.8b-mini-instruct-4k-q4_K_M
 ```
 
-Note that unpack includes an implicit pull. You can also only unpack parts of a ModelKit which can make cloning much faster. Learn more in our [Next Steps](./next-steps.md) guide.
-
-Now start your LLM dev server locally:
+Now start your LLM dev server locally using the [dev start command](./cli/cli-reference.md#kit-dev-start):
 
 ```sh
-kit dev .
+kit dev start .
 ```
 
-In the command output you'll see a URL you can use to interact with the LLM. You can control parameters of the model, change the prompt, or chat with the LLM.
+In the command output you'll see a URL you can use to interact with the LLM (there's a command flag if you want to always use the same port). You can control parameters of the model, change the prompt, or chat with the LLM.
+
+If you need to get logs use the [dev logs command](./cli/cli-reference.md#kit-dev-logs):
+
+```sh
+kit dev logs
```

Comment:
aren't these flags instead of attributes? like
```
kit dev --logs
```

and/or

```
kit dev --stop
```

@gorkem @amisevsk  ?